halyard-deploy/halyard-deploy.gradle

```diff
@@ -7,4 +7,6 @@ dependencies {
 
   compile project(':halyard-config')
   compile project(':halyard-core')
+
+  compile 'com.google.apis:google-api-services-storage:v1-rev72-1.22.0'
```

Comment:
Can probably just get this from here now: https://github.com/spinnaker/spinnaker-dependencies/blob/master/src/spinnaker-dependencies.template#L23